### PR TITLE
Added GPT-3 bot template and renamed a faas function for better naming consistency

### PR DESCRIPTION
**Pull request by cshape**

### CODE REVIEW

**Comment by cshape** [comment]
If a reviewer without a GPT-3 key wants to test it out, let me know and I can share mine.

**Review by dahby — COMMENTED** [review]
@cshape Looks good so far, my biggest suggestion would be to remove your credentials prior to exporting the bot. When importing the bot into my account, a reference to your credentials shows. It isn't usable, but still shows an error and could cause some confusion. 

Screen Shot 2021-08-31 at 3 45 49 PM

File: bot-samples/GPT3-template/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# Requirements
+
+- A [GPT-3 developer key](https://beta.openai.com/)
+- A conversational cloud account
+
+# Getting Started Steps
+
+1. Import the "GPT3 Character Template" bot from this repo into your Conversational Cloud account
+2. In the bot's "GPT3" integration, add a Credential for GPT-3. This should be an "Access Token" type credential. The access token will be the one GPT-3 developer key provided by OpenAI, and the token type should be "Bearer"
```
Comment (dahby) [inline bot-samples/GPT3-template/README.md:13]: @cshape Let's include a link to our documentation on creating credentials here for ease of access.

**Comment by cshape** [comment]
@dahby Thanks for the review! I've updated the PR to reflect your comments.

**Comment by JasonSowers** [comment]
> @cshape Looks good so far, my biggest suggestion would be to remove your credentials prior to exporting the bot. When importing the bot into my account, a reference to your credentials shows. It isn't usable, but still shows an error and could cause some confusion.
> 
> Screen Shot 2021-08-31 at 3 45 49 PM

if you used a valid credential it will show in the commit history, so please revert those commits or start a new branch and apply your changes.  If the credentials was not valid don't worry about it.

**Comment by cshape** [comment]
> > @cshape Looks good so far, my biggest suggestion would be to remove your credentials prior to exporting the bot. When importing the bot into my account, a reference to your credentials shows. It isn't usable, but still shows an error and could cause some confusion.
> > Screen Shot 2021-08-31 at 3 45 49 PM
> 
> if you used a valid credential it will show in the commit history, so please revert those commits or start a new branch and apply your changes. If the credentials was not valid don't worry about it.

That's a great point, @JasonSowers. Luckily, the credential seems to be hashed in the JSON so I think it's all good.

**Review by dahby — APPROVED** [review]
Looks good! Merging

**Pull request merged**